fiwaremurano/meta/io.murano/Resources/linux-init.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+service murano-agent stop
+
+AgentConfigBase64='%AGENT_CONFIG_BASE64%'
+
+if [ ! -d /etc/murano ]; then
+    mkdir /etc/murano
```

Comment:
Cloud-init is executed by root user. Anyway, this code belongs to the oficial reporitory